docs/data/sdks/browser-2/index.md

```diff
@@ -18,6 +18,9 @@ The Browser SDK lets you send events to Amplitude. This library is open-source,
 !!!note "Migration guide"
     This is the documentation for the Amplitude Browser SDK 2.0. If you are using the Browser SDK 1.0, refer to the migration documentation: [Browser SDK 2.0 Migration Guide](/data/sdks/browser-2/migration/). If you are using the maintenance SDK, refer to the migration documentation: [Browser SDK Migration Guide](/data/sdks/typescript-browser/migration/).
 
+--8<-- "includes/size/browser.md"
+    For example you can search `@amplitude/analytics-browser@2.2.3`.
```

Comment:
It seems like we can link to the package without a version. I think it would be nice to add the links here e.g.
```suggestion
    For example you can search [`@amplitude/analytics-browser`](https://bundlephobia.com/package/@amplitude/analytics-browser) at a specific version.
```